src/game_api/script/usertypes/vanilla_render_lua.cpp

```diff
@@ -138,7 +138,7 @@ auto g_angle_style = CORNER_FINISH::ADAPTIVE;
 
 void VanillaRenderContext::set_corner_finish(CORNER_FINISH c)
 {
-    g_angle_style = c;
+    g_angle_style = c; // can i make this per lua environment instead of global?
```

Comment:
Would make more sense i think

Reply:
I guess make this function set it in an optional field in the calling backend, then make VanillaRenderContext constructor get it from the calling backend?

Reply:
It also could be a field in the VanillaRenderContext if you wanted

Reply:
Isn't `VanillaRenderContext` created per callback or I'm misunderstanding something?

Reply:
But there is no point in using this function more then once. I can't imagine someone willing to have multiple stuff drawn with different "style". I created it just in case someone would be complaining that he is trying to draw very sharp angle but the outside corner looks weird or extends too far